src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php

```diff
@@ -73,13 +73,18 @@ public function downloadAction(Request $request, $id)
             $version = $request->get('v', null);
             $noCount = $request->get('no-count', false);
 
+            $dispositionType = 'attachment';
+            if ($request->get('inline', false)) {
```

Comment:
how can this be set in the twig? because i think there is only one url to the action without this parameter! at least we should add a handler where it can be passed!